### PR TITLE
[NetteToSymfony] Kdyby Events Doctrine event subscriber to Symfony Doctrine one

### LINKED ISSUE #1045: [NetteToSymfony] Doctrine Event Subscirber implementing Symfony
```diff
+use Doctrine\Common\EventSubscriber;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;

-final class DispatchDomainEventsSubscriber implements EventSubscriberInterface
+final class DispatchDomainEventsSubscriber implements EventSubscriber
{
-     public static function getSubscribedEvents(): array
+    public function getSubscribedEvents(): array
    {
        return [
-            Events::postPersist => 'keepAggregateRoots',
-            Events::postUpdate => 'keepAggregateRoots',
-            Events::postRemove => 'keepAggregateRoots',
-            Events::postFlush => 'dispatchDomainEvents',
+            Events::postPersist,
+            Events::postUpdate,
+            Events::postRemove,
+            Events::postFlush,
        ];
    }

+	public function postPersist(LifecycleEventArgs $event): void
+	{
+		$this->keepAggregateRoots($event);
+    }
+
+
+	public function postUpdate(LifecycleEventArgs $event): void
+	{
+		$this->keepAggregateRoots($event);
+    }
+
+
+	public function postRemove(LifecycleEventArgs $event): void
+	{
+		$this->keepAggregateRoots($event);
+   }
+
+
+	public function postFlush(): void
+	{
+		$this->dispatchDomainEvents();
+    }

}
```

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #1045

### CODE REVIEW

**Comment by TomasVotruba** [comment]
@JanMikes Hey, I made the Doctrine Kdyby event subscriber migrator to Symfony Doctrine event. Could you check it, if it's correct? :)

**Comment by JanMikes** [comment]
Okay, im going for it!

> Kdyby event subscriber migrator to Symfony Doctrine event
Btw, this sentence is confusing, as we discussed in person, Symfony events and Doctrine events are 2 very different things (though they have similar method names and concepts) 😄

File: packages/NetteToSymfony/tests/Rector/Class_/KdybyEventDoctrineSubscriberToDoctrineSubscriberRector/Fixture/fixture.php.inc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rector\NetteToSymfony\Tests\Rector\Class_\KdybyEventDoctrineSubscriberToDoctrineSubscriberRector\Fixture;
+
+use Kdyby\Doctrine\Events;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class SomeDoctrineEventSubscriber implements EventSubscriberInterface
```
Comment (JanMikes) [inline packages/NetteToSymfony/tests/Rector/Class_/KdybyEventDoctrineSubscriberToDoctrineSubscriberRector/Fixture/fixture.php.inc]: According to the [documentation](https://github.com/Kdyby/Events/blob/master/docs/en/index.md#doctrine-events), the interface to implement is `Kdyby\Events\Subscriber`, might this be an issue or is it okay?

**Comment by TomasVotruba** [comment]
> Btw, this sentence is confusing, as we discussed in person, Symfony events and Doctrine events are 2 very different things (though they have similar method names and concepts) 😄

What could be better description? :)

File: packages/NetteToSymfony/tests/Rector/Class_/KdybyEventDoctrineSubscriberToDoctrineSubscriberRector/Fixture/fixture.php.inc (hunk 1 above)
Comment (TomasVotruba) [reply packages/NetteToSymfony/tests/Rector/Class_/KdybyEventDoctrineSubscriberToDoctrineSubscriberRector/Fixture/fixture.php.inc]: I probably incorrectly copied the code in #1045

I'll change it, no problem. Thanks

Comment (TomasVotruba) [reply packages/NetteToSymfony/tests/Rector/Class_/KdybyEventDoctrineSubscriberToDoctrineSubscriberRector/Fixture/fixture.php.inc]: The format is completelly differnt than in the test :D I'm lost.

Could you add the proppre change example in diff? Or link to entrydo code would help, I could not find it.

**Comment by TomasVotruba** [comment]
Resolved due to very specific use case

**Pull request closed without merging**